snekbox/nsjail.py

```diff
@@ -188,7 +189,12 @@ def python3(
             py_args: Arguments to pass to Python.
             files: FileAttachments to write to the sandbox prior to running Python.
             nsjail_args: Overrides for the NsJail configuration.
+            binary_path: The path to the binary to execute under.
+                If None, defaults to "/lang/python/default/bin/python"
         """
+        if not binary_path:
+            binary_path = "/lang/python/default/bin/python"
```

Comment:
Had to do something slightly idfferent so that hte default cna be used both by calling python3 directly, and when calling the API https://github.com/python-discord/snekbox/pull/196/commits/66c6d27335a5d1fc23bd0889d299cb79f3418219